extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/exec/MSQInsertTest.java

```diff
@@ -1079,7 +1074,7 @@ public void testInsertLimitWithPeriodGranularityThrowsException()
                              + "LIMIT 50 "
                              + "PARTITIONED BY MONTH")
                      .setExpectedValidationErrorMatcher(CoreMatchers.allOf(
-                         CoreMatchers.instanceOf(SqlPlanningException.class),
+                         CoreMatchers.instanceOf(DruidException.class),
                          ThrowableMessageMatcher.hasMessage(CoreMatchers.containsString(
                              "INSERT and REPLACE queries cannot have a LIMIT unless PARTITIONED BY is \"ALL\""))
```

Comment:
Yes, it could.  I'm not sure which, but you are either seeing stuff left behind when I first started converting tests and hadn't yet build the Matcher to make it simpler (and I missed it in going back to simplify). OR, with fixing the conflicts after 2 months dormant, I will admit that I didn't have all of the context still in my head and perhaps did something a bit rote.  Either way, it's much simpler to switch to the `invalidSqlContains()`.